eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -183,6 +188,14 @@ abstract class SafeMojo extends AbstractMojo {
         )
     );
 
+    /**
+     * EO cache directory.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    @Parameter(property = "eo.cache")
+    @SuppressWarnings("PMD.ImmutableField")
+    protected Path cache = Paths.get(System.getProperty("user.home")).resolve(".eo");
+
```

Comment:
@Graur `cache` attribute is here is `SafeMojo`

Reply:
@potatmen Of course, you added it in this pull request. I'm trying to figure out why we can't use the same structure that our Mojos already have? By placing this parameter in `AssembleMojo` and `OptimizedMojo`

Reply:
@Graur according to this comment https://github.com/objectionary/eo/pull/1293#discussion_r985379456